### DISCUSSION TITLE
Welcome to Discussions of the Flood project!

### DISCUSSION THREAD

**Discussion by jesec** [jesec/flood#117] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by sryazanov** [comment]
How to move files (set location) on touch screen devices (iPad in particular)?

Context menu required right-click, which is not available. Long tap doesn’t work.

**Reply by jesec** [reply]
Sorry for the late reply. Apparently I missed the email, and I only saw this after randomly navigated around the project. 

Long press should work. I can do that on my iPhone.
